api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/ConceptTranslatorImpl.java

```diff
@@ -105,10 +119,7 @@ public Concept toOpenmrsType(@Nonnull CodeableConcept concept) {
 	private void addConceptCoding(Coding coding, String system, String code, Concept concept) {
 		coding.setSystem(system);
 		coding.setCode(code);
-		
-		ConceptName conceptName = concept.getName();
-		String display = (conceptName == null || conceptName.getName() == null) ? "" : conceptName.getName();
-		coding.setDisplay(display);
+		coding.setDisplay(concept.getDisplayString());
```

Comment:
The above is handled by concept.getDisplayString() in a standard way.  It also allows for easier mocking.